topics/transcriptomics/tutorials/scrna-JUPYTER-trajectories/tutorial.md

```diff
@@ -0,0 +1,216 @@
+>
+{: .hands_on}
+
+## Filtering for T-cells
+
+One problem with our current dataset is that it's not just T-cells: we found in the previous tutorial that it also contains macrophages and red blood cells. This is a problem, because trajectory analysis will generally try to find relationships between all the cells in the sample. We need to remove those cell types to analyse the trajectory.
+
+> ### {% icon hands_on %} Hands-on: Removing macrophages and RBCs
+>
+> 1. {% tool [Manipulate AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate/anndata_manipulate/0.7.5+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: `Final cell annotated object`
+>    - *"Function to manipulate the object"*: `Filter observations or variables`
+>    - *"What to filter?"*: `Observations (obs)`
+>    - *"Type of filtering?"*: `By key (column) values`
+>    - *"Key to filter"*: `cell_type`
+>    - *"Type of value to filter"*: `Text`
+>    - *"Filter"*: `not equal to`
+>    - *"Value"*: `RBC`
+>
+> 2. {% tool [Manipulate AnnData](toolshed.g2.bx.psu.edu/repos/iuc/anndata_manipulate/anndata_manipulate/0.7.5+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: (output of **Manipulate AnnData** {% icon tool %})
+>    - *"Function to manipulate the object"*: `Filter observations or variables`
+>    - *"What to filter?"*: `Observations (obs)`
+>    - *"Type of filtering?"*: `By key (column) values`
+>    - *"Key to filter"*: `cell_type`
+>    - *"Type of value to filter"*: `Text`
+>    - *"Filter"*: `not equal to`
+>    - *"Value"*: `Macrophages`
+>
+> 3. **Rename** {% icon galaxy-pencil %} output h5ad `T-cell_object.h5ad`
+>
+{: .hands_on}
+
+You should now have `7795` cells, as opposed to the `7874` you started with. You've only removed a few cells (the contaminants!), but it makes a big difference in the next steps.
+
+Take note of what # this dataset is in your history, as you will need that shortly!
+
+## Launching Jupyter
+
+> ### {% icon hands_on %} Hands-on: Downloading the directions
+>
+> 1. Click the **Trajectories_Instructions.ipynb** dataset and the download {% icon galaxy-save %} button to get the file onto your computer
+>
+{: .hands_on}
+
+JupyterLab is a bit like RStudio but for other coding languages. What, you've never heard of RStudio? Then don't worry, just follow the instructions!
+
+{% icon warning %} Please note: this is only currently available on the [usegalaxy.eu](https://usegalaxy.eu) site.
+
+> ### {% icon hands_on %} Hands-on: Launching JupyterLab
+>
+> 1. {% tool [Interactive Jupyter Notebook](tool_id=interactive_tool_jupyter_notebook) %} with the following parameters:
+>    - *"Do you already have a notebook?"*: `Start with a fresh notebook`
+>    - {% icon param-file %} *"Include data into the environment"*:  `Nothing selected`
+>
+>  This may take a moment, but once the `Executed notebook` in your dataset is orange, you are up and running!
```

Comment:
```suggestion
>    This may take a moment, but once the `Executed notebook` in your dataset is orange, you are up and running!
```

I think this will reset numbering with the current indentation level.